server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -119,6 +122,124 @@ const orderMutations = {
       return models.Order.findOne(query);
     },
   },
+  updateOrder: {
+    type: Order,
+    description: "Update an Order's amount, tier, payment method, or frequency",
+    args: {
+      order: {
+        type: new GraphQLNonNull(OrderReferenceInput),
+        description: 'Object matching the OrderReferenceInput (id)',
+      },
+      amount: {
+        type: GraphQLInt,
+        description: 'Amount in cents of the order',
+      },
+      frequency: {
+        type: GraphQLString,
+        description: 'Frequency of the recurring order, either MONTHLY or YEARLY',
+      },
+      tier: {
+        type: GraphQLString,
+        description: 'The tier of the recurring contribution, like Backer or Sponsor',
+      },
+      paymentMethod: {
+        type: PaymentMethodReferenceInput,
+        description: 'Object matching the PaymentMethodReferenceInput (uuid)',
```

Comment:
I don't get that description. Could just be: "Reference to a Payment Method to update the order with".